Nova/src/main/kotlin/xyz/xenondevs/nova/tileentity/impl/mob/MobDuplicator.kt

```diff
@@ -139,6 +137,11 @@ class MobDuplicator(
     private fun spawnEntity() {
         val entity = if (keepNbt) EntityUtils.deserializeAndSpawn(entityData!!, spawnLocation, NBTUtils::removeItemData).bukkitEntity
         else spawnLocation.world!!.spawnEntity(spawnLocation, entityType!!)
+
+        val nmsEntity = entity.nmsEntity
+        if (NERF_MOBS && nmsEntity is net.minecraft.world.entity.Mob)
```

Comment:
Why is net.minecraft.world.entity.Mob not imported?

Reply:
whoops